src/components/OSCALSystemImplementation.js

```diff
@@ -31,6 +31,140 @@ const OSCALSystemImplementationSubDataHeader = styled(TableCell)`
   padding: 0.75em 0.75em;
 `;
 
+const OSCALSystemImplementationTableTitle = styled(Typography)`
+  flex: 1 1 100%;
+`;
+
+function PropertiesTable(props) {
+  return (
+    <TableContainer>
+      <Table size="small">
+        <TableBody>
+          {props.list &&
+            props.list.map((property) => (
+              <TableRow key={property.name}>
+                <OSCALSystemImplementationSubDataHeader
+                  component="th"
+                  scope="row"
+                >
+                  {property.name}
+                </OSCALSystemImplementationSubDataHeader>
+                <SmallTableCell>{property.value}</SmallTableCell>
+              </TableRow>
+            ))}
+        </TableBody>
+      </Table>
+    </TableContainer>
+  );
+}
+
+function OSCALSystemImplementationComponents(props) {
+  return (
+    <>
+      <OSCALSystemImplementationTableTitle variant="h6" id="tableTitle">
+        Components
+      </OSCALSystemImplementationTableTitle>
+      <TableContainer>
+        <Table aria-label="Components">
+          <TableHead>
+            <TableRow>
+              <TableCell>Component</TableCell>
+              <TableCell>Type</TableCell>
+              <TableCell>Status</TableCell>
+              <TableCell>Properties</TableCell>
+              <TableCell>Responsible Roles</TableCell>
+            </TableRow>
+          </TableHead>
+          <TableBody>
+            {Object.entries(props.components).map(([key, component]) => (
+              <TableRow key={key}>
+                <ComponentTableCell component="th" scope="row">
+                  <StyledTooltip title={component.description}>
+                    <Typography variant="body2">{component.title}</Typography>
+                  </StyledTooltip>
```

Comment:
The `Typography` tag is supposed to be a `markup-line` and the `title` should be a `markup-multiline`. The same goes for the `Users` table. This was not done in #156, so there is not precedent that I can see. `ToolTip` does not allow you to have the `Markdown` component as a child.

Reply:
As far as I know, `title` is _never_ multi-line in OSCAL. It is always markup-line. |
Since this appears to be about the users table in the SSP, I am looking now at that construct (`/system-security-plan/system-implementation/user/title` and it is `markup-line`, not multi-line.
Please double-check the spec.

Reply:
The title of the `ToolTip` is the `user description` here, which is supposed to be `markup-multiline` I believe. The fact that is is the `title` prob is confusing.

Reply:
`user title` is the `child` component of the `Tooltip`. Which is supposed to be  `markup-line`. The issue is that the `ToolTip` does not play nice with our `OSCALMarkdown` components.

Reply:
With that said, how strongly do we feel about supporting `Markdown` here? Since we are close to the end of the sprint, it may be worth pushing to another issue. I will continue working on it until we decide otherwise.

Reply:
This is being pushed to #592